aiomysql/connection.py

```diff
@@ -465,6 +495,22 @@ def _connect(self):
             self._next_seq_id = 0
 
             yield from self._get_server_information()
+            if self._sslcontext and not self._unix_socket:
+                packet = make_auth_ssl(
+                        charset=charset_by_name(self._charset).id,
+                        client_flags=(self.client_flag ^ 2048))
+                self.write_packet(packet)
+                # upgrade connection to ssl
+                # close recent reader and write and keep socket connected
+                sock = self._writer._transport._sock
+                # patch asyncion
+                self._writer._transport._call_connection_lost = \
```

Comment:
could you clarify why we need this line? `_call_connection_los` is private... and may not work for other `loop` implementations.